pkg/cmd/create/create_cluster_gke.go

```diff
@@ -292,9 +322,11 @@ func (o *CreateClusterGKEOptions) createClusterGKE() error {
 			machineType = defaultMachineType
 			log.Logger().Infof(util.QuestionAnswer("Defaulting to machine type", machineType))
 		}
+	} else {
+		log.Logger().Infof(util.QuestionAnswer("Configured to machine type", machineType))
 	}
 
-	minNumOfNodes := o.Flags.MinNumOfNodes
+	minNumOfNodes := viper.GetString("min-num-nodes")
```

Comment:
I would suggest that we parse the config in a central place. It is possible to define all fields in a struct and the use the `viper.Unmarshal` instead of fetching each value one by one through the code. Here is an example https://gist.github.com/chazcheadle/45bf85b793dea2b71bd05ebaa3c28644. 

This has the advantage that we can validate the fields early on and provide sensitive defaults from the very beginning.